### DISCUSSION TITLE
ProgressBar Ticks and Orientation

### DISCUSSION THREAD

**Discussion by boematic** [AvaloniaUI/Avalonia#8828] [General]
I'm using a ProgressBar with Orientation=Vertical.  When I turn on ShowProgressText the Text is flipped.  How can I change the orientation of the Text?  Also, how do I enable Ticks on the progress bar with values?  Thanks.

**Comment by timunie** [comment]
Have a look into the original Styles. For the Ticks, I think you need to create your own ControlTemplate. 

The Style for current Fluent Theme: https://github.com/AvaloniaUI/Avalonia/blob/stable/0.10.x/src/Avalonia.Themes.Fluent/Controls/ProgressBar.xaml
